test/controllers/api/v1/developments_controller_test.rb

```diff
@@ -37,6 +43,12 @@ def development
     end
   end
 
+  test 'should handle empty params with grace and poise' do
```

Comment:
Thank you. This was the only thing I was looking for in this pull request. This will allow me to significantly reduce complexity in the client-side code.